src/app/components/LoadingLayer/LoadingLayer.jsx

```diff
@@ -8,34 +9,37 @@ class LoadingLayer extends React.Component {
     }
 
     return (
-      <div
-        className="loadingLayer"
-        role="alertdialog"
-        aria-labelledby="loading-animation"
-        aria-describedby="loading-description"
-        aria-live="assertive"
-        aria-atomic="true"
-        ref={this.props.childRef}
-      >
-        <div className="loadingLayer-layer"></div>
-        <div className="loadingLayer-texts">
-          <span id="loading-animation" className="loadingLayer-texts-loadingWord">
-            Loading...
-          </span>
-          <span
-            id="loading-description"
-            className="loadingLayer-texts-title"
-          >
-            {this.props.title}
-          </span>
-          <div className="loadingDots">
-            <span></span>
-            <span></span>
-            <span></span>
-            <span></span>
+      <FocusTrap className="focus-trap">
```

Comment:
If you want to, you can remove the outer `div` with the className loadingLayer and the other attributes and move it to `FocusTrap`. It's not necessary though but the docs says it should work.

Reply:
yeah good idea. I will update it.